packages/tokens/src/typography.ts

```diff
@@ -219,6 +227,11 @@ export function getWebTextVariants() {
       description: 'code',
       value: textVariants.code,
     },
+
+    address: {
```

Comment:
Usually we match these exactly to the design system but I think this is a nice place to put this also. I wonder does it need to be in the global typography though as it's only really used in address-displayer?

We have this other long standing task to clean up typography on mobile, maybe you'd like to investigate that as you were working on this? 

https://github.com/leather-io/mono/issues/72

It's unclear to me what exactly needs to be done there or if it's done already but it would be nice to close it off if we can

Reply:
I've been initially reluctant about adding this to the typography tokens too, but hardcoding in two places felt even messier. Already asked the design team to see if there are any other use-cases for this so we can come up with a more generic style name.

Re: #72, I've actually set out to refactor the typography a little bit, but it started leaking out of the scope of this task very quickly, so I'm planning to do that separately some time soon. Might be a good place to figure out the `address` as a style conundrum too.

Reply:
Sounds great! I just mentioned https://github.com/leather-io/mono/issues/72 as it would be good to keep that in mind and find out what exactly else needs to be done there if you are refactoring the typography. 

I agree lets not do it here 👍